content/en/docs/releases.md

```diff
@@ -76,19 +74,78 @@ Or you can also specify releases with independent configurations:
 ```
 You can build specific releases using `rebar3 release -n <release_name>`
 
-## Developing
+## Modes
 
-While developing you'll likely want all your changes to applications be immediately available in the release. `relx` provides `dev_mode` for this. Instead of copying the applications that make up the release to the release structure it creates symlinks, so compiling and restarting or loading the changed modules, is all that is necessary.
+Other modes include `prod` and `minimal`:
 
 ```erlang
 {relx, [...
-        {dev_mode, true},
+        {mode, <mode>},
         ...
        ]
 }.
+```
+
+| Mode | Expanded Options |
+| --------------------- | ------------------------------------------------------------- |
+| `dev` | `[{dev_mode, true}, {include_src, true}, {debug_info, keep}, {include_erts, false}]` |
+| `prod`  | `[{include_src, false}, {debug_info, strip}, {include_erts, true}, {dev_mode, false}]` |
+| `minimal` | `[{include_src, false}, {debug_info, strip}, {include_erts, false}, {dev_mode, false}]` |
+
+While developing you'll likely want all your changes to applications be
+immediately available in the release. `relx` provides multiple modes, including
+a mode `dev` for this particular use case. Instead of copying the applications
+that make up the release to the release structure it creates symlinks, so
+compiling and restarting or loading the changed modules, is all that is necessary.
+
+The `prod` mode expands to options that are commonly used for production
+releases: `include_src` is `false` so source code is not included in the release,
+`debug_info` is set to `strip` which makes BEAM files smaller and only removes
+data that is used by tools you more than likely aren't including in your
+release, and sets `include_erts` to `true` to bundle in the current Erlang
+runtime, making a release you can copy to a compatible target and run without
+first installing Erlang.
+
+The `minimal` mode is the same as `prod` except it does not include the Erlang
+runtime.
+
+You can override options that the modes expand to by including explicit setting
+them. For example, if you did want to keep the debug info in the BEAM modules
+then you can use a configuration like:
+
+```erlang
+[
+  {mode, prod},
+  {debug_info, keep}
+]
+```
+
+## Verification Checks
+
+### Missing Functions
+
+The `exref` option to relx will utilize
```

Comment:
`exref` in my opinion since that is the systools option. If we end up having to build our own to make it do what we want we would name that `xref` I guess.